k8s/consul/Makefile

```diff
@@ -1,21 +1,27 @@
 include ../crd.Makefile
 include ../gcloud.Makefile
 include ../var.Makefile
+include ../images.Makefile
 
 APP_ID ?= consul
 
 SOURCE_REGISTRY ?= marketplace.gcr.io/google
 
-# The digest should be updated automatically, or tag <major>.<minor>.<path/build>
-# should be available on GCP Marketplace.
-# Use `.build/consul/VERSION` target to fill the RELEASE variable.
-CONSUL_TAG ?= sha256:225fa1caac9e6a10ea126d20cef26a50b2e7dc4be0bfda37ba0f1bcd3b412019
-CONSUL_EXPORTER_TAG ?= sha256:1db79fa556f3c6cd23ef382d041b8473a7d947fdcaf36533994df52cb97d50ec
+TRACK ?= 1.4
 METRICS_EXPORTER_TAG ?= v0.5.1
 
+# List of images used in application
+TARGETS := consul-exporter prometheus-to-sd
+
+# Image names should correspond with TARGET_IMAGES list
+image-$(APP_ID) ?= $(call get_sha256,$(SOURCE_REGISTRY)/consul1:$(TRACK))
+image-consul-exporter ?= $(call get_sha256,$(SOURCE_REGISTRY)/consul1:exporter)
+image-prometheus-to-sd ?= $(call get_sha256,k8s.gcr.io/prometheus-to-sd:$(METRICS_EXPORTER_TAG))
+
+C2D_CONTAINER_RELEASE ?= $(call get_var_from_container,$(image-$(APP_ID)),CONSUL_VERSION)
```

Comment:
Should be `C2D_RELEASE` instead?

Reply:
Yes. But we don't have updated container yet.

Reply:
I added TODO to update Makefile to use C2D_RELEASE variable once it will be present in container.